commandLine/options.go

```diff
@@ -159,3 +162,19 @@ func createWallet(cmd *cobra.Command, args []string) {
 
 	fmt.Printf("New address is: %s\n", address)
 }
+
+func createPublisher(cmd *cobra.Command, args []string) {
+	nodeInfo := utils.GetNodeInfo()
+	if nodeInfo == nil {
+		fmt.Println("'port.pid' file already exists, cannot create a publisher")
+
+	} else {
+		fmt.Println("port.pid file exists, proceeding to start publisher node.")
+		network.P2p(true) 
+	}
+}
```

Comment:
The if - else statement should be:
If the file port.pid already exist, the function shouldn't call the network.P2p function.
If the file port.pid doesn't exist, the function should create a publisher node.